Index: 05-TierverwaltungSystem/05-TierverwaltungSystem.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/05-TierverwaltungSystem/05-TierverwaltungSystem.iml b/05-TierverwaltungSystem/05-TierverwaltungSystem.iml
new file mode 100644
--- /dev/null	(date 1741877302966)
+++ b/05-TierverwaltungSystem/05-TierverwaltungSystem.iml	(date 1741877302966)
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
\ No newline at end of file
Index: 04-Kunden_u_Betreuer/src/KundenBetreuung1.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/04-Kunden_u_Betreuer/src/KundenBetreuung1.java b/04-Kunden_u_Betreuer/src/KundenBetreuung1.java
new file mode 100644
--- /dev/null	(date 1741875083702)
+++ b/04-Kunden_u_Betreuer/src/KundenBetreuung1.java	(date 1741875083702)
@@ -0,0 +1,52 @@
+import java.io.IOException;
+import java.util.Scanner;
+
+public class KundenBetreuung1 {
+    public static void main(String[] args) throws IOException {
+
+        Scanner scanner = new Scanner(System.in);
+        //Print Titel
+        System.out.println("\n----------------------------------------");
+        System.out.println("KundenBetreuung");
+        System.out.println("----------------------------------------\n");
+        //Arrays für Klassenobjekte erzeugen
+        Kunde[] k = new Kunde[5];
+        Betreuer[] b = new Betreuer[5];
+        //Initialize Kunden in Arrays
+        k[0] = new Kunde(1,"Lukas Schneider",0,0);
+        k[1] = new Kunde(2,"Hannah Fischer",0,0);
+        k[2] = new Kunde(3,"Paul Wagner",0,0);
+        //Initialize Betreuer in Arrays
+        b[0] = new Betreuer(4,"Sophie Becker",0);
+        b[1] = new Betreuer(5,"Jonas Lehmann",0);
+        b[2] = new Betreuer(6,"Marie Hoffmann",0);
+        //Kunden Betreuer zuweisen
+        k[0].setKundenBetreuer(b[0]);
+        k[1].setKundenBetreuer(b[1]);
+        k[2].setKundenBetreuer(b[2]);
+        //KundeErteiltAuftrag
+        System.out.println("\tAnwendungsfälle\n");
+        System.out.println("\t1.Kunde erteilt Auftrag\n");
+        System.out.println("\t2.Kunde bezahlt für den Auftrag\n");
+        System.out.println("\t3.Betreuer ermittelt Provision\n");
+        System.out.println("\t4.Auftrag wird beendet\n");
+        System.out.println("Wählen sie aus: ");
+        int userInput = scanner.nextInt();
+        switch (userInput){
+            case 1:
+                Auftrag(k,b);
+                break;
+            case 2:
+                Bezahlung(k,b);
+                break;
+            case 3:
+                Provision(b);
+                break;
+            case 4:
+                break;
+            default:
+                System.out.println("Ungültige eingabe!");
+                break;
+        }
+    }
+}
Index: 04-Kunden_u_Betreuer/src/Bezahlung.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/04-Kunden_u_Betreuer/src/Bezahlung.java b/04-Kunden_u_Betreuer/src/Bezahlung.java
new file mode 100644
--- /dev/null	(date 1741875347094)
+++ b/04-Kunden_u_Betreuer/src/Bezahlung.java	(date 1741875347094)
@@ -0,0 +1,7 @@
+import java.io.IOException;
+
+public class Bezahlung {
+    public static void Bezahlung(Kunde[ ] k, Betreuer [ ] b) throws IOException {
+
+    }
+}
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ProjectRootManager\" version=\"2\" languageLevel=\"JDK_21\" default=\"true\" project-jdk-name=\"openjdk-21\" project-jdk-type=\"JavaSDK\">\r\n    <output url=\"file://$PROJECT_DIR$/out\" />\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	(revision b338796d716aafdce3acdee50c3d59a554cf3c78)
+++ b/.idea/misc.xml	(date 1741868597783)
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_21" default="true" project-jdk-name="openjdk-21" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_23" default="true" project-jdk-name="openjdk-23" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/out" />
   </component>
 </project>
\ No newline at end of file
Index: .idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ProjectModuleManager\">\r\n    <modules>\r\n      <module fileurl=\"file://$PROJECT_DIR$/01-Autovermietung/01-Autovermietung.iml\" filepath=\"$PROJECT_DIR$/01-Autovermietung/01-Autovermietung.iml\" />\r\n      <module fileurl=\"file://$PROJECT_DIR$/02-Kassenumsaetze/02-Kassenumsaetze.iml\" filepath=\"$PROJECT_DIR$/02-Kassenumsaetze/02-Kassenumsaetze.iml\" />\r\n      <module fileurl=\"file://$PROJECT_DIR$/03-Firmenumsatz/03-Firmenumsatz.iml\" filepath=\"$PROJECT_DIR$/03-Firmenumsatz/03-Firmenumsatz.iml\" />\r\n      <module fileurl=\"file://$PROJECT_DIR$/.idea/Anwendungsfälle.iml\" filepath=\"$PROJECT_DIR$/.idea/Anwendungsfälle.iml\" />\r\n    </modules>\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
--- a/.idea/modules.xml	(revision b338796d716aafdce3acdee50c3d59a554cf3c78)
+++ b/.idea/modules.xml	(date 1741877302966)
@@ -5,6 +5,8 @@
       <module fileurl="file://$PROJECT_DIR$/01-Autovermietung/01-Autovermietung.iml" filepath="$PROJECT_DIR$/01-Autovermietung/01-Autovermietung.iml" />
       <module fileurl="file://$PROJECT_DIR$/02-Kassenumsaetze/02-Kassenumsaetze.iml" filepath="$PROJECT_DIR$/02-Kassenumsaetze/02-Kassenumsaetze.iml" />
       <module fileurl="file://$PROJECT_DIR$/03-Firmenumsatz/03-Firmenumsatz.iml" filepath="$PROJECT_DIR$/03-Firmenumsatz/03-Firmenumsatz.iml" />
+      <module fileurl="file://$PROJECT_DIR$/04-Kunden_u_Betreuer/04-Kunden_u_Betreuer.iml" filepath="$PROJECT_DIR$/04-Kunden_u_Betreuer/04-Kunden_u_Betreuer.iml" />
+      <module fileurl="file://$PROJECT_DIR$/05-TierverwaltungSystem/05-TierverwaltungSystem.iml" filepath="$PROJECT_DIR$/05-TierverwaltungSystem/05-TierverwaltungSystem.iml" />
       <module fileurl="file://$PROJECT_DIR$/.idea/Anwendungsfälle.iml" filepath="$PROJECT_DIR$/.idea/Anwendungsfälle.iml" />
     </modules>
   </component>
Index: 04-Kunden_u_Betreuer/src/Auftrag.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/04-Kunden_u_Betreuer/src/Auftrag.java b/04-Kunden_u_Betreuer/src/Auftrag.java
new file mode 100644
--- /dev/null	(date 1741875347077)
+++ b/04-Kunden_u_Betreuer/src/Auftrag.java	(date 1741875347077)
@@ -0,0 +1,7 @@
+import java.io.IOException;
+
+public class Auftrag {
+    public static void Auftrag(Kunde[ ] k, Betreuer [ ] b) throws IOException {
+
+    }
+}
Index: 04-Kunden_u_Betreuer/src/Betreuer.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/04-Kunden_u_Betreuer/src/Betreuer.java b/04-Kunden_u_Betreuer/src/Betreuer.java
new file mode 100644
--- /dev/null	(date 1741871730672)
+++ b/04-Kunden_u_Betreuer/src/Betreuer.java	(date 1741871730672)
@@ -0,0 +1,38 @@
+public class Betreuer {
+
+    private int betreuerNr;
+    private String betreuerName;
+    private double betreuerProvision;
+
+    public Betreuer(int carerNr,String carerName,double carerProvision){
+        this.betreuerNr = carerNr;
+        this.betreuerName = carerName;
+        this.betreuerProvision = carerProvision;
+    }
+
+    public int getBetreuerNr() {
+        return betreuerNr;
+    }
+
+    public void setBetreuerNr(int betreuerNr) {
+        this.betreuerNr = betreuerNr;
+    }
+
+    public String getBetreuerName() {
+        return betreuerName;
+    }
+
+    public void setBetreuerName(String betreuerName) {
+        this.betreuerName = betreuerName;
+    }
+
+    public double getBetreuerProvision() {
+        return betreuerProvision;
+    }
+
+    public void setBetreuerProvision(double betreuerProvision) {
+        this.betreuerProvision = betreuerProvision;
+    }
+
+    public void betreuerProvision(){}
+}
Index: 04-Kunden_u_Betreuer/src/Kunde.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/04-Kunden_u_Betreuer/src/Kunde.java b/04-Kunden_u_Betreuer/src/Kunde.java
new file mode 100644
--- /dev/null	(date 1741872713034)
+++ b/04-Kunden_u_Betreuer/src/Kunde.java	(date 1741872713034)
@@ -0,0 +1,56 @@
+public class Kunde {
+
+    private int kundenNr;
+    private String kundenName;
+    private Betreuer kundenBetreuer;
+    private double kundenUmsatz;
+    private double kundenOffen;
+
+    public Kunde(int customerNr,String customerName,double customerRevenue,double customerOpen){
+        this.kundenNr = customerNr;
+        this.kundenName = customerName;
+        this.kundenUmsatz = customerRevenue;
+        this.kundenOffen = customerOpen;
+    }
+
+    public int getKundenNr() {
+        return kundenNr;
+    }
+
+    public void setKundenNr(int kundenNr) {
+        this.kundenNr = kundenNr;
+    }
+
+    public String getKundenName() {
+        return kundenName;
+    }
+
+    public void setKundenName(String kundenName) {
+        this.kundenName = kundenName;
+    }
+
+    public Betreuer getKundenBetreuer() {
+        return kundenBetreuer;
+    }
+
+    public void setKundenBetreuer(Betreuer kundenBetreuer) {
+        this.kundenBetreuer = kundenBetreuer;
+    }
+
+    public double getKundenUmsatz() {
+        return kundenUmsatz;
+    }
+
+    public void setKundenUmsatz(double kundenUmsatz) {
+        this.kundenUmsatz = kundenUmsatz;
+    }
+
+    public double getKundenOffen() {
+        return kundenOffen;
+    }
+
+    public void setKundenOffen(double kundenOffen) {
+        this.kundenOffen = kundenOffen;
+    }
+    public void kundeBezahlt(){}
+}
Index: 04-Kunden_u_Betreuer/04-Kunden_u_Betreuer.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/04-Kunden_u_Betreuer/04-Kunden_u_Betreuer.iml b/04-Kunden_u_Betreuer/04-Kunden_u_Betreuer.iml
new file mode 100644
--- /dev/null	(date 1741870687627)
+++ b/04-Kunden_u_Betreuer/04-Kunden_u_Betreuer.iml	(date 1741870687627)
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
\ No newline at end of file
Index: 04-Kunden_u_Betreuer/src/Provision.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/04-Kunden_u_Betreuer/src/Provision.java b/04-Kunden_u_Betreuer/src/Provision.java
new file mode 100644
--- /dev/null	(date 1741875347065)
+++ b/04-Kunden_u_Betreuer/src/Provision.java	(date 1741875347065)
@@ -0,0 +1,7 @@
+import java.io.IOException;
+
+public class Provision {
+    public static void Provision(Betreuer [ ] b) throws IOException {
+
+    }
+}
